VultisigApp/VultisigApp/Views/TransactionMemo/TransactionMemoVerifyView.swift

```diff
@@ -29,18 +32,53 @@ struct TransactionMemoVerifyView: View {
         }
     }
     
-    var view: some View {
-        container
-    }
-    
     var content: some View {
-        VStack {
-            fields
-            button
+        VStack(spacing: 0) {
+            ScrollView {
+                fields
+            }
+            Spacer()
+            VStack(spacing: 16) {
+                if tx.isFastVault {
```

Comment:
<img width="1159" alt="image" src="https://github.com/user-attachments/assets/6ce858f1-dfe0-43f0-b8ac-42da5bac4aff">

The UI looks weird , can you please make sure both button stay at the bottom , and close to each other , like  the following
Also , can you rename it Sign -> "Pair Sign"
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/e35bf909-d894-47b5-bd6a-1b76def3ea25">

Reply:
Fixed
<img width="1263" alt="image" src="https://github.com/user-attachments/assets/2bbb5742-12cc-4a4b-9fda-787b1cd42ae7">